outreach/v0.21_announcement.md

```diff
@@ -0,0 +1,29 @@
+<!---
+ When 0.21.0 is released this announcement will be posted to r/rust and the
+ rust user forum. Also post in rust audio discord: https://discord.gg/8qW6q2k
+-->
+
+# Announcing rodio 0.21
+
+
+Rodio is an audio playback library. It can decode audio files, synthesize new
+sounds, apply effects to sounds & mix them. Rodio has been part of the Rust
+ecosystem for 9 years now! 🎉.
+
+## New release
+Its been 5 months since our last release. Since the our team has grown to 3
+maintainers! Thank you Petr and Roderick! And a big thanks for the countless
+other contributors helping out. Thanks to them this release:
+
+- Makes the API easier to use
+    - Its now impossible to accidentally drop the `Outputstream`.
+    - Our types are no longer generic over sample type.
+- The decoders have been improved enabling *gapless* playback and offering many
```

Comment:
1. Remove the point about gapless playback, that was already the case in v0.20 and before when using Symphonia.
2. Add that `total_duration` is now available.

Reply:
we already had total duration right? (0.20.1 docs: https://docs.rs/rodio/latest/rodio/decoder/struct.Decoder.html#method.total_duration)

Reply:
Yes, I meant that the Symphonia decoder now returns `Some(total_duration)` when it can. Previously it always returned `None`.